### PR TITLE
Crypto heatmaps

### PR DESCRIPTION
**Pull request by jose-donato**
# Description

Adds command `hm` to overview crypto menu that displays a heat-map of top cryptocurrencies (similar to [https://finviz.com/map.ashx](https://finviz.com/map.ashx)).

image

Can easily be converted to stocks context.

`squarify`, the dependency used to generate the heat-map, was added to `pypoetry.toml`

Thank you for the help @DidierRLopes 🙌 

# How has this been tested?
Just tested running locally. @Chavithra can you explain to me how we are testing commands that are only displaying charts?

# Checklist:

- [x] Update [our Hugo documentation](https://gamestonkterminal.github.io/GamestonkTerminal/) following [these guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/website).
- [ ] Update our tests following [these guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/blob/main/CONTRIBUTING.md).
- [x] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/scripts).

# Others
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] My code passes all the checks pylint, flake8, black, ... To speed up development you should run `pre-commit install`.
- [ ] New and existing unit tests pass locally with my changes. You can test this locally using `pytest tests/...`.

### CODE REVIEW

File: gamestonk_terminal/cryptocurrency/overview/overview_controller.py [hunk 1]
```diff
@@ -131,6 +136,9 @@ def __init__(self, queue: List[str] = None):
             choices["cpcontracts"]["-s"] = {
                 c: None for c in coinpaprika_model.CONTRACTS_FILTERS
             }
+            choices["hm"] = {c: None for c in get_categories_keys()}
+            choices["hm"]["--category"] = {c: None for c in get_categories_keys()}
```
Comment (DidierRLopes) [inline gamestonk_terminal/cryptocurrency/overview/overview_controller.py:140]: didn't we decide to just have the main one, e.g. `hm something` on autocomplete. To not add delay to all commands? Or did I just misremembered that?

**Review by DidierRLopes — CHANGES_REQUESTED** [review]
You need to add squarify to requirements.txt, requirements-full.txt and poetry.lock

**Review by jmaslek — CHANGES_REQUESTED** [review]
This also isn't using the new plotting convention.  Should have external_axes and import the custom theme.

File: gamestonk_terminal/cryptocurrency/overview/overview_controller.py (hunk 1 above)
Comment (jose-donato) [reply gamestonk_terminal/cryptocurrency/overview/overview_controller.py:140]: my bad. forgot about this

**Comment by jose-donato** [comment]
@DidierRLopes @jmaslek everything updated

**Review by DidierRLopes — APPROVED** [review]

**Comment by jmaslek** [comment]
One more request 😁 can we capitalize in the box and show what the color represents, i.e

```
BTC
+69%
```

**Comment by jose-donato** [comment]
> One more request 😁 can we capitalize in the box and show what the color represents, i.e
> 
> ```
> BTC
> +69%
> ```

yep! thought about that also. will try to do it

**Comment by DidierRLopes** [comment]
@jmaslek stamp approval needed here

**Comment by jose-donato** [comment]
> @jmaslek stamp approval needed here

it's my bad sorry, still need to try what @jmaslek suggested, will try it now

**Comment by jose-donato** [comment]
> One more request 😁 can we capitalize in the box and show what the color represents, i.e
> 
> ```
> BTC
> +69%
> ```

image
way better :)

**Review by jmaslek — APPROVED** [review]

**Pull request merged**